### ISSUE TITLE
Clay %da .^ doesn't resolve until the next file revision

### ISSUE THREAD

**Issue by ohAitch** [urbit/arvo#76]
```
> .^(arch %cy /==(scot %da now)/stub)
[fil=~ dir={}]
>> [%camping %c %y [p=~zod q=%home r=[%da p=~2016.3.16..20.31.23..56aa.0000.0000.0001]] s=/stub]
> .^(arch %cy /==(scot %da +(now))/stub)
{{block}}
```
attempting to interact with the console has predictable results
```
%dy-no-prompt
%dy-no-prompt
```
until some clay write happens
```
: /~zod/home/719/web/md
[fil=~ dir={}]
~zod:dojo>
```

A far cry from `~s0..0000.0000.0000.0001`

**Comment by galenwp** [comment]
You have a number of bugs on this subject.  Basically, you'd like to be able to multi-task in the dojo.  I get it, but can't we basically just put these all under a single issue?

**Comment by ohAitch** [comment]
No, the "blocked dojo" printf is just being used to demonstrate what's going on: clay is accepting a subscription for the future date, and then doesn't respond on that subscription when the date occurs; it later does if a revision happens. If you did this inside e.g. kiln, using an explicit clay request, I expect you'd have the same problem.

**Comment by belisarius222** [comment]
To clarify, it seems that the issue here is that Clay should be setting a timer when asked for a file at a date in the future. The timer expiration should cause Clay to respond to this request, as opposed to the current behavior where Clay only responds to the request when a file changes. This is not a Dojo issue.
